features/add_dishes.feature

```diff
@@ -1,24 +1,35 @@
-Feature: Add dishes to menu
-  As a user
-  In order to decide from various dishes from a restaurant
-  I would like to see many dishes in the restaurant´s menu
+Feature: Add dishes to restaurant
+  As a restaurant owner
+  In order to manage my menu
+  I would like to be able to add dishes on my menu
```

Comment:
Okay, so I'm NOT found of changing the user story on existing features. I would prefer you create a new feature file. The question is why was the feature written the way it was before. Seems weird to me. Anyway seems like the original story was about displaying dishes and your is about adding them. Why was it named `add_dishes.feature` originally. (Please check git blame and ask for explanation)